Formula/g/gaul.rb

```diff
@@ -33,10 +33,13 @@ def install
     # Run autoreconf on macOS to rebuild configure script so that it doesn't try
     # to build with a flat namespace.
     system "autoreconf", "--force", "--verbose", "--install" if OS.mac?
-    system "./configure", "--disable-dependency-tracking",
-                          "--disable-debug",
-                          "--disable-g",
-                          "--prefix=#{prefix}"
+
+    args = ["--disable-g"]
+
+    # Help old config scripts identify arm64 linux
+    args << "--build=aarch64-unknown-linux-gnu" if OS.linux? && Hardware::CPU.arm? && Hardware::CPU.is_64_bit?
```

Comment:
Lots of formulae are affected by this. Maybe we should just add it to `std_configure_args`.

Reply:
yeah, that is not a bad idea (but this only applies to the old configure scripts though)

Reply:
I kind of think of this as libtool configure patches we did before.

Reply:
In this case for this formula, we already do autoreconf on macOS anyway so I'd just make it the same for both.

Reply:
yeah, that is true.